README.md

```diff
@@ -1,119 +1,116 @@
+connector to become operational the `runtime` needs to perform several important tasks (="bootstrapping"). For an
+example take a look at
+[this runtime](launchers/basic/src/main/java/org/eclipse/dataspaceconnector/runtime/ConnectorRuntime.java)
 
-- KeyVault
-- Blob Store Account
-- an AKS cluster
-- two app registrations / service principals
+### `data-protocols`
 
-App registrations are used to authenticate apps against Azure AD and are secured with certificates, so before
-provisioning any resources, the Genesis Script will generate a certificate (interactively) and upload it to the app
-registrations.
+Contains implementations for communication protocols a connector might use, such as IDS.
 
-The script requires Azure CLI (`az`) and `jq` (a JSON processor) and you need to be logged in to Azure CLI. Once that's
-done, simply `cd scripts` and invoke
+### `samples`
 
-```bash
-./genesis.sh <optional-prefix>
-```
+Contains code that demonstrates how the connector can be used in various scenarios. For example, it shows how to run a
+connector from a unit test in order to try out functionality quickly or how to implement an outward-facing REST API for
+a connector.
 
-the `<optional-prefix>` is not necessary, but if specified it should be a string without special characters, as it is
-used as resource suffix in the Azure resources. If omitted, the current Posix timestamp is used.
+### `common`
 
-After having completed its work, which could take >10mins, the scripts can automatically cleanup resources, please
-observe the cli output.
+Contains utility code such as collection utils, string utils and helper classes.
 
-_Note that the Genesis Script does not deploy any applications such as Nifi, this is handled in a second stage!_
+### `scripts`
 
-## Contributing
+Contains several scripts to deploy a connector in an AKS cluster on Microsoft Azure using Terraform.
+
+## Roadmap
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License
-Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For
-details, visit https://cla.opensource.microsoft.com.
+[TBW]
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide a CLA and decorate
-the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only
-need to do this once across all repos using our CLA.
+## Contributing
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+Please find the contributing guidelines here [TBW]
 
 ## Trademarks
```

Comment:
While it might be legitimate (due to the initial contribution of Microsoft), referring explicitly and only to Microsoft within this section could be confusing for the readers. It could be implied that this project is owned by Microsoft.

Additionally, I am not aware of other OSS repos referring to trademark policies.

Reply:
personally I dont' see any problem with removing it, but i'd rather not step on anyone's toes here. In order to get this PR merged quickly: can we leave it in for now an potentially remove it later?

Reply:
Ok, let's not get blocked by this and keep it for now.